pkg/certmanageroperator/install.go

```diff
@@ -65,7 +65,14 @@ func installOperator(t test.TestHelper) {
 
 func waitOperatorSucceded(t test.TestHelper, certManagerOperatorNs string) {
 	t.Log("Waiting for cert-manager-operator to succeed")
-	oc.WaitFor(t, certManagerOperatorNs, "csv", certmanagerVersion, "jsonpath='{.status.phase}'=Succeeded")
+	// When the operator is installed, the CSV take some time to be created, need to wait until is created to validate the phase
+	retry.UntilSuccess(t, func(t test.TestHelper) {
+		if !certManagerOperatorExists(t) {
+			t.Fatalf("cert-manager-operator is not yet installed")
```

Comment:
Does `UntilSuccess` work with `Fatal`? I mean, will it be retried or the test will fail in the first iteration?

Reply:
Yes, it works with the fatal. For example:

```
install.go:62: STEP 2: Install cert-manager-operator
    install.go:67:    Waiting for cert-manager-operator to succeed
    install.go:71:    FAILURE: cert-manager-operator is not yet installed
    install.go:69:    --- Attempt 1/60 failed. Retrying...
    install.go:71:    FAILURE: cert-manager-operator is not yet installed
    install.go:69:    --- Attempt 2/60 failed. Retrying...
    install.go:71:    FAILURE: cert-manager-operator is not yet installed
    install.go:69:    --- Attempt 3/60 failed. Retrying...
    install.go:71:    FAILURE: cert-manager-operator is not yet installed
    install.go:69:    --- Attempt 4/60 failed. Retrying...
    install.go:69:    --- Attempt 5/60 successful; total time: 7.58s
```